scripts/test/lld.py

```diff
@@ -63,7 +37,8 @@ def test_wasm_emscripten_finalize():
       if ext != '.out' and not os.path.exists(expected_file):
         continue
 
-      cmd = WASM_EMSCRIPTEN_FINALIZE + [wast_path, '-S'] + ext_args
+      cmd = (WASM_EMSCRIPTEN_FINALIZE +
+             [wast_path, '-S', '--global-base=1024'] + ext_args)
```

Comment:
Can we rely on the default for --global-base rather than passing it here and above?

Reply:
I'd rather not, we should test that this explicitly sets the value.

... Which means I should explicitly pass something that ISN'T the default to better verify it works.

Reply:
Does emscripten support the non-default path though?  Can the user control this?
If so, then yes I agree should use and test the flag.

Reply:
Yep and yep https://github.com/kripken/emscripten/pull/6215/files#diff-e64ed58dc838bc8cb95295fef5447e1fR1896